ote/src/clj/ote/services/routes.clj

```diff
@@ -19,11 +19,21 @@
             [cheshire.core :as cheshire]
             [ote.db.tx :as tx]
             [jeesql.core :refer [defqueries]]
-            [ote.environment :as environment])
+            [ote.environment :as environment]
+            [ote.db.feature])                               ; specql table definitions
   (:import (org.postgis PGgeometry Point Geometry)))
 
 (defqueries "ote/services/routes.sql")
 
+(defn- service-state-response [db]
```

Comment:
It is private function and only related to this service but still wondering if it should be renamed something like 'valid-route-service-state?'

Reply:
I see what your going for. However,
Logic is naming is the same as in admin service side: function returns a request response, hence name should have "-response" suffix. "get-route-response" in the file follows the same pattern.
Suffix "?" is for boolean return values.